x-pack/plugin/inference/qa/inference-service-tests/src/javaRestTest/java/org/elasticsearch/xpack/inference/InferenceCrudIT.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+package org.elasticsearch.xpack.inference;
+
+import org.elasticsearch.inference.TaskType;
+
+import java.io.IOException;
+import java.util.List;
+import java.util.Map;
+
+import static org.hamcrest.Matchers.hasSize;
+
+public class InferenceCrudIT extends InferenceBaseRestTest {
+
+    @SuppressWarnings("unchecked")
+    public void testGet() throws IOException {
+        for (int i = 0; i < 5; i++) {
+            putModel("se_model_" + i, mockServiceModelConfig(), TaskType.SPARSE_EMBEDDING);
+        }
+        for (int i = 0; i < 4; i++) {
+            putModel("te_model_" + i, mockServiceModelConfig(), TaskType.TEXT_EMBEDDING);
+        }
+
+        var getAllModels = (List<Map<String, Object>>) getModels("_all", TaskType.ANY).get("models");
```

Comment:
I would call `getModels("_all", null)` here, which under the hood calls `/_inference/_all`

Reply:
I added a `getAllModels()` function to use here